src/main/markdown/gettingstarted-v2.md

```diff
@@ -1,7 +1,7 @@
 Getting Started
 ===
 
-Starting with 2.11.0 of the GWT SDK the [webAppCreator](gettingstarted-outdated.html) is deprecated. Instead, we will use 
+Starting with 2.11.0 of the GWT SDK the [webAppCreator](gettingstarted.html) is deprecated. Instead, we will use
```

Comment:
note that there is no `-outdated` file - perhaps there was at one point, but it never merged.

Reply:
might be an refactoring issue.